### PR TITLE
docs(file): implement and document attrs and vars

### PR DESCRIPTION
**Pull request by BY00565233**
## Description

`<hx-file>` : implement and document attributes and variables

### What are the relevant story cards/tickets? Any additional PRs or other references?

Jira: SURF-2107

## Before you request a review for this PR:

- [x] For UI changes, did you manually test in recent versions of modern browsers (Chrome, Firefox, and Safari)?
- [x] For UI changes, did you manually test in IE11 and legacy Edge?
- [ ] Did you add component tests for any new code?
- [ ] Did you run the component unit tests via `yarn test` to ensure all tests passed?
- [ ] Did you include a screenshot of the component tests?
- [ ] If you changed/added functionality, did you update the demo page and documentation?
- [ ] If needed, did you add or modify the demo test page to test the changed/added functionality?
- [x] Did you assign reviewers?
- [x] In Jira, have you linked to this PR on the ticket(s)?

### CODE REVIEW

**Review by 100stacks — CHANGES_REQUESTED** [review]
### @BY00565233, please check and verify these updates work in IE11 and Legacy Edge along with Firefox, Safari, and Chrome.

**Comment by BY00565233** [comment]
> ### @BY00565233, please check and verify these updates work in IE11 and Legacy Edge along with Firefox, Safari, and Chrome.

@100stacks I verified and updated the code to work in all 5 browsers . please review

File: src/scss/components/file/_hx-file-tile.scss [hunk 1]
```diff
@@ -1,17 +1,21 @@
...
 
   &[progress] {
     color: $gray-600;
+    color: var(--hxFile-hxFileTile-progress-color, $gray-600);
   }
 
   &[invalid] {
-    border-color: $red-900;
+    border-color: $red-status-900;
+    border-color: var(--hxFile-hxFileTile-invalid-border-color, $red-status-900);
```
Comment (100stacks) [inline src/scss/components/file/_hx-file-tile.scss:19]: Please change [invalid] status colors to `$red-status-500`.

**Review by 100stacks — CHANGES_REQUESTED** [review]

**Review by lalithkarikelli — APPROVED** [review]
DEV LGTM

**Review by 100stacks — APPROVED** [review]
Dev LGTM

**Pull request merged**